db/payment_icons.yml

```diff
@@ -2206,3 +2206,7 @@
   name: orangemobilemoney
   label: Orange Mobile Money
   group: wallets
+-
+  name: sistecredito
+  label: SISTECREDITO
```

Comment:
Is there a reason why uppercase is used here? If not, please change to sentence case.